src/routers/TabRouter.js

```diff
@@ -62,6 +62,15 @@ export default (
       let state = inputState;
       if (!state) {
         const routes = order.map((routeName: string) => {
+          let accessibilityLabel;
```

Comment:
Is there any way to change the accessibility label programmatically at run-time? It looks like the accessibilityLabels are set when the initial tab routes are set up.